models/user.go

```diff
@@ -719,6 +727,17 @@ func (u *User) IsMailable() bool {
 	return u.IsActive
 }
 
+// EmailNotifications returns the User's email notification preference
+func (u *User) EmailNotifications() string {
+	return u.EmailNotificationsPreference
+}
+
+// SetEmailNotifications sets the user's email notification preference
+func (u *User) SetEmailNotifications(set string) {
+	u.EmailNotificationsPreference = set
```

Comment:
This is not a good usage, we should update user by the specific columns.

Reply:
Okay, updating specific columns now.